packages/camera/camera_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.10+2
+
+* Don't set the FPS range unless video recording. It can cause dark image previews on some devices becuse the auto exposure algorithm is more constrained after fixing a min & max FPS value. 
```

Comment:
Can you update this description to include the behavior change to users of this plugin that rely on the defaults? (even if that is no impact)

Reply:
@reidbaker Would this be more clear?

Don't set the FPS range unless video recording. It can cause dark image previews on some devices becuse the auto exposure algorithm is more constrained after fixing the min/max FPS range to the same value. This change has the side effect that providing the `fps` parameter will not affect the camera preview when not video recording. And if you need a lower frame rate in your image streaming handler, you can skip frames by checking the time it passed since the last frame.

Reply:
That looks great.